packages/components/breadcrumb/_breadcrumb.scss

```diff
@@ -15,11 +15,10 @@
  */
 
 .nhsuk-breadcrumb {
-  @include print-hide();
+  @include print-hide(); /* [1] */
```

Comment:
This [1] comment here would seem to relate to the bespoke spacing, and the [2] down below next to the padding relates  to the print hiding. i.e. the wrong way round.

I'm not entirely sure why we even do comments this way - it seems to completely separate them from the code they're referencing and make them really hard to read. Is it time to break away from this habit?

Also, the mixin is literally called `print-hide` - I don't even think a comment adds anything at all, we could remove that one.

Reply:
Taking the padding example - I see that the [1] can be put on two lines, and the comment relating to those lines can be "applied" to both - I guess that's the argument for this comment style? But if you were to just put that comment on one of the padding lines, or even above it like this:

```
// Bespoke spacing numbers used as there is no 12px spacing mapped in settings/spacing.
padding-bottom: nhsuk-spacing(3);
padding-top: 20px;
```

that would seem just as clear, and keeps the comment near the code?

Sorry to nitpick on this. It may be a change for another day. It was the numbers being the wrong way round that drew my attention to it! Maybe just sort the numbers for this change and we can bikeshed about the comments separately?

Reply:
Agree that [1] and [2] should be swapped.

Also, [1] message should read "Bespoke spacing number used as there is no 20px spacing mapped in settings/spacing."
Also, [6] and [7] comments need to be removed as there is no reference to them anymore.

And agree that discussions about comments should be left for another day ;-)

Reply:
Good catch, cheers @andymantell and @mcheung-nhs. I've updated the comments and the corresponding references.

I agree about the detachment from the comment and the code; it can be easy to overlook and sometimes confusing. Ha, the `print-hide` is overkill. Maybe we address these issues at the next frontend catchup?